mev-commit-cli.sh

```diff
@@ -131,6 +131,14 @@ deploy_contracts() {
     echo "Waiting for Geth POA network to be fully up..."
     sleep 10
 
+    # Deploy create2 proxy from alpine container
+    docker run --rm -v \
+        "$GETH_POA_PATH/geth-poa/util/deploy_create2.sh:/deploy_create2.sh" \
+        alpine /bin/sh -c \
+        "apk add --no-cache curl jq \
+        && chmod +x /deploy_create2.sh \
+        && /deploy_create2.sh http://localhost:8545"
```

Comment:
I believe localhost should be sl-bootnode and for that to be resolved you also need to pass in --rm --network like z

```   
docker run --rm --network "$DOCKER_NETWORK_NAME" \
        -e RPC_URL="$rpc_url" \
        -e CHAIN_ID="$chain_id" \
        -e PRIVATE_KEY="$private_key" \
        contract-deployer
 ```